test/plugin/scenarios/kafka-scenario/src/main/java/test/org/apache/skywalking/apm/testcase/kafka/controller/CaseController.java

```diff
@@ -68,12 +68,10 @@ public String kafkaCase() {
         wrapProducer(producer -> {
             ProducerRecord<String, String> record = new ProducerRecord<String, String>(topicName, "testKey", Integer.toString(1));
             record.headers().add("TEST", "TEST".getBytes());
-            producer.send(record, new Callback() {
-                @Override
-                public void onCompletion(RecordMetadata metadata, Exception exception) {
-                    logger.info("send success metadata={}", metadata);
-                }
-            });
+            Callback callback = (metadata, exception) -> {
+                logger.info("send success metadata={}", metadata);
+            };
+            producer.send(record, callback);
```

Comment:
As you have changed the scenario, you should change the expected data file, otherwise, all tests break.